uaclient/entitlements/landscape.py

```diff
@@ -0,0 +1,166 @@
+import logging
+import os
+import subprocess
+from typing import Any, Dict, Optional, Tuple
+
+from uaclient import apt, event_logger, exceptions, messages, system, util
+from uaclient.entitlements.base import UAEntitlement
+from uaclient.entitlements.entitlement_status import ApplicationStatus
+
+event = event_logger.get_event_logger()
+
+LANDSCAPE_CLIENT_PACKAGE_NAME = "landscape-client"
+LANDSCAPE_CLIENT_CONFIG_PATH = "/etc/landscape/client.conf"
+LANDSCAPE_CLIENT_CONFIG_PATH_DISABLE_BACKUP = (
+    "/etc/landscape/client.conf.pro-disable-backup"
+)
+
+
+class LandscapeEntitlement(UAEntitlement):
+    name = "landscape"
+    title = "Landscape"
+    description = "Management and administration tool for Ubuntu"
+    help_doc_url = "https://ubuntu.com/landscape"
+
+    def _perform_enable(self, silent: bool = False) -> bool:
+        event.info(messages.APT_UPDATING_LISTS)
+        apt.run_apt_update_command()
+
+        event.info(
+            messages.INSTALLING_PACKAGE.format(LANDSCAPE_CLIENT_PACKAGE_NAME)
+        )
+        apt.run_apt_install_command([LANDSCAPE_CLIENT_PACKAGE_NAME])
```

Comment:
Shouldn't the package be tied to the contract server product definition ? Or are you trying to avoid the creation of a `requiredPackages` attribute on their side ?

Reply:
I was treating it more like how we treat livepatch.

We could move the package name into the resource definition, but then we'd still probably have the command name hard-coded, so I'm not sure if its worth it. What do you think?

Reply:
I guess that's similar to anbox though. I can look into what it would take to make a `requiredPackages` attribute work

Reply:
I'm starting to implement this and I think it makes sense, I'll put up a new commit to see the differences soon

Reply:
Just pushed a commit implementing this